extension/src/Xliff/XLIFFDocument.ts

```diff
@@ -1096,7 +1120,11 @@ export interface XliffDocumentInterface {
   targetLanguage?: string;
   original?: string;
   transunit?: TransUnit[];
-  toString(replaceSelfClosingTags: boolean, formatXml: boolean): string;
+  toString(
```

Comment:
idea: This could be a good opportunity to replace the parameters with an object, defined by an interface :)

Reply:
just too much work to fix this right now. Quite a few references on `toString()`. 😅